civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -63,6 +67,7 @@
               data: function () {
                 return data;
               },
+              isCachedContacts: isContactsCached,
```

Comment:
name things consistently. One is `isCachedContacts`, the other one is `isContactsCached`, i don't see the reason for this difference